launch/tier4_localization_launch/launch/localization.launch.xml

```diff
@@ -6,10 +6,13 @@
   <arg name="random_downsample_filter_param_path"/>
   <arg name="ndt_scan_matcher_param_path"/>
   <arg name="localization_error_monitor_param_path"/>
+  <arg name="eagleye_param_path"/>
 
   <arg name="input/pointcloud" default="/sensing/lidar/top/outlier_filtered/pointcloud" description="The topic will be used in the localization util module"/>
   <arg name="use_pointcloud_container" default="true" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
+  <arg name="localization_mode" default="lidar" description="select localization mode. lidar, gnss"/>
+  <arg name="use_eagleye_twist" default="false" description="Mode to correct wheel speed and IMU by GNSS Doppler velocity"/>
```

Comment:
Conditional branch based on these two parameters (`localization_mode` and `use_eagleye_twist`) seems a bit too complicated...

I think this issue is due to the incompatibility between Autoware localization architecture (which treats pose estimator and twist estimator as individual modules) and Eagleye (which couples pose and twist domain, if I understood correctly). And I am sorry I cannot come up with a good solution for now.

Do you have any ideas how to address this issue?

Reply:
How about something like this?
Eagleye as pose_estimator and eagleye as twist_estimator are activated separately.
The disadvantage is that the number of overlapping nodes increases.
https://github.com/autowarefoundation/autoware.universe/pull/2848/commits/9f007da4af9d0191707e8ce3f3c77c6426660896

Reply:
@YamatoAndo @kminoda (@OsamuSekino @Aoki-Takanose)
What do you think about this idea?

Reply:
I have corrected the problem with reference to the above PR.

commit
https://github.com/autowarefoundation/autoware.universe/pull/2848/commits/ba6a71db16068adf6be88f2f497c241b88f38598